library/Potoki/Core/Prelude.hs

```diff
@@ -82,3 +84,54 @@ import Control.Concurrent.STM as Exports
 -- acquire
 -------------------------
 import Acquire.Acquire as Exports
+
+-- base-prelude
```

Comment:
No need for "base-prelude" in this package, since we already are reexporting all the "base" modules from the local Prelude